src/main/scala/org/globalforestwatch/layers/Layer.scala

```diff
@@ -237,6 +237,7 @@ trait RequiredILayer extends RequiredLayer with ILayer {
   def fetchWindow(windowKey: SpatialKey,
                   windowLayout: LayoutDefinition): ITile = {
     val layoutSource = LayoutTileSource.spatial(source, windowLayout)
+    println(s"Fetching required int tile ${source.dataPath.value}, key ${windowKey}")
```

Comment:
Make sure to revert these print statements as your final commit of the PR.

Reply:
Or can I comment them out so that they're easy to use in the future?